script/scaleUpMesh/scaleUpMesh.py

```diff
@@ -0,0 +1,100 @@
+import regex
+
+
+class Node:
+    idx: int
+    x: float
+    y: float
+    z: float
+
+    def __init__(self, idx: int, x: float, y: float, z: float):
+        self.idx = idx
+        self.x = x
+        self.y = y
+        self.z = z
+
+    @staticmethod
+    def from_line(line):
+        results = regex.search(r"^(\d+)\s([+-]?[0-9]*\.[0-9]+(?:e-?\d+)?)\s([+-]?[0-9]*\.[0-9]+(?:e-?\d+)?)\s([+-]?[0-9]*\.[0-9]+(?:e-?\d+)?)", line)
+        if results is None:
+            raise RuntimeError(f"Regex parsing failed for line: <{line}>")
+        return Node(int(results.group(1)), float(results.group(2)), float(results.group(3)), float(results.group(4)))
+
+    def __add__(self, other):
+        return Node(-1, self.x + other.x, self.y + other.y, self.z + other.z)
+
+
+class Face:
+    idx: int
+    vIdx1: int
+    vIdx2: int
+    vIdx3: int
+
+    def __init__(self, idx: int, vIdx1: int, vIdx2: int, vIdx3: int):
+        self.idx = idx
+        self.vIdx1 = vIdx1
+        self.vIdx2 = vIdx2
+        self.vIdx3 = vIdx3
+
+    def midPoint(self, nodes):
+        node = nodes[self.vIdx1] + nodes[self.vIdx2] + nodes[self.vIdx3]
+        node.idx = len(nodes)
+        return node
+
+    def generateSplitFaces(self, nodeIdx, nextFaceIdx):
+        return [Face(nextFaceIdx, self.vIdx1, self.vIdx2, nodeIdx), Face(nextFaceIdx + 1, self.vIdx2, self.vIdx3, nodeIdx), Face(nextFaceIdx + 2, self.vIdx3, self.vIdx1, nodeIdx)]
+
+
+    @staticmethod
+    def from_line(line):
+        results = regex.search(r"^(\d+)\s(\d+)\s(\d+)\s(\d+)", line)
+        if results is None:
+            return None
+        return Face(int(results.group(1)), int(results.group(2)), int(results.group(3)), int(results.group(4)))
+
+
+def fetchData():
```

Comment:
In Java/ C++ one often uses `CamelCase`, for Python, however, the standard recommends `snake_case`.

Actually, even standardized these days in PEP 8: https://peps.python.org/pep-0008/#function-and-variable-names

For this script, you don't need to change this anymore. Just keep it in mind for future scripts.